fms/models/llama.py

```diff
@@ -403,8 +403,11 @@ def factory(**kwargs):
 models.register_model(_architecture_name, "13b", _llama_factory_factory(_13b_config))
 models.register_model(_architecture_name, "70b", _llama_factory_factory(_70b_config))
 
-_convert_fused_qkv_0_0_4 = serialization.simple_mapping_adapter(
-    [serialization._legacy_attn_unfused_to_fused_weight_conversion]
+__v0_0_4_adapter = serialization.simple_mapping_adapter(
```

Comment:
hmm... at some point we should probably rename "simple_mapping_adapter" since no longer simply mapping individual parameters to new parameter names